scripts/check_compatibility.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
this is very nice!

Reply:
Thanks 👍 It's nice to have safeguards for introducing potential issues this way.